app/views/shipit/stacks/_banners.html.erb

```diff
@@ -53,9 +53,9 @@
   <div class="banner">
     <div class="banner__inner wrapper">
       <div class="banner__content">
-        <h2 class="banner__title">Continuous Delivery Delayed!</h2>
+        <h2 class="banner__title">Continuous delivery delayed</h2>
```

Comment:
We use sentence case in headings in the product, this should probably be the case here as well (see Polaris content guidelines).